src/homepageExperience/components/steps/SampleAppCard.tsx

```diff
@@ -57,10 +57,14 @@ const SampleAppCard: FC<Props> = ({wizardEventName, handleNextStepEvent}) => {
       {resources.map(app => (
         <ResourceCard
           className="homepage-wizard-next-steps"
-          onClick={() => handleNextStepEvent(wizardEventName, app.title)}
           key={app.links[wizardEventName]}
         >
-          <SafeBlankLink href={app.links[wizardEventName]}>
+          <SafeBlankLink
+            href={app.links[wizardEventName]}
+            onClick={() =>
+              handleNextStepEvent(wizardEventName, app.title.replace(/ +/g, ''))
```

Comment:
there's a tested utility function [`normalizeEventName`](https://github.com/influxdata/ui/blob/master/src/cloud/utils/reporting.ts#L152) that will probably do this for you

Reply:
that's exactly what i need